File: Frontend/src/app/DTOs/Group/GroupInfoDTO.ts
import { LearnzFileFrontendDTO } from "../File/LearnzFileFrontendDTO";
import { GroupInfoMemberDTO } from "./GroupInfoMemberDTO";

export interface GroupInfoDTO {
  groupId: string;
  name: string;
  description: string;
  profileImage: LearnzFileFrontendDTO;
  members: GroupInfoMemberDTO[];
  isUserAdmin: boolean;
}

File: Backend/DTOs/GroupInfoDTO.cs
namespace Learnz.DTOs;
public class GroupInfoDTO
{
    public Guid GroupId { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public LearnzFileFrontendDTO ProfileImage { get; set; }
    public List<GroupInfoMemberDTO> Members { get; set; }
    public bool IsUserAdmin { get; set; }
}
